src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_BY_THIS_STRING = "\\W+";
+    private static final int EMPTY_STRING_LENGTH = 0;
+    private final StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to divide the program logic into 3 methods:
1. Reading data from a file
2. Create a report from this data
3. Writing a report to a file

And you method getStatistic() should only call these three methods